src/main/java/chess/dao/ChessDAO.java

```diff
@@ -0,0 +1,23 @@
+package chess.dao;
+
+import chess.domain.Board;
+import chess.domain.Team;
+import chess.domain.piece.Piece;
+import chess.domain.position.Position;
+
+public interface ChessDAO {
```

Comment:
```suggestion
public interface ChessDao {
```

취향 차이일 수 있는데, 저는 카멜케이스를 사용하는 편 입니다.
만약 HttpXmlDao과 같이 축약어가 연속되는 클래스의 경우 어떻게 네이밍 할 수 있을지 고민해보아요.